R/parameters.R

```diff
@@ -151,35 +180,40 @@
 #' * phi_indoors - proportion of bites taken indoors; default = 0.90
 #'
 #' treatment parameters:
-#' please set treatment parameters with the convenience functions in
-#' `drug_parameters.R`
+#' please set drug and treatment parameters with the convenience functions
+#' `set_drugs` and `set_clinical_treatment`
 #'
 #' * drug_efficacy - a vector of efficacies for available drugs; default = turned off
 #' * drug_rel_c - a vector of relative onward infectiousness values for drugs; default = turned off
 #' * drug_prophylaxis_shape - a vector of shape parameters for weibull curves to
 #' model prophylaxis for each drug; default = turned off
 #' * drug_prophylaxis_scale - a vector of scale parameters for weibull curves to
 #' model prophylaxis for each drug; default = turned off
-#' * clinical_treatment_drugs - a vector of drugs that are available for
-#' clinically diseased (these values refer to the index in drug_* parameters); default = NULL, NULL, NULL
-#' * clinical_treatment_coverage - a vector of coverage values for each drug; default = NULL, NULL, NULL
+#' * clinical_treatment_drugs - a list of drugs that are available for
+#' clinically diseased (these values refer to the index in drug_* parameters); default = NULL
+#' * clinical_treatment_timesteps - a list of vectors giving timesteps at which the
+#' clinical treatment coverage changes (these values refer to the index in drug_* parameters); default = NULL
+#' * clinical_treatment_coverage - a list of vectors giving coverage values for each drug; default = NULL
 #'
 #' PEV parameters: 
```

Comment:
Just a note while I remeber - some interventions won't be available for vivax, so we will need a catch in the relevant set intervention functions to stop them being implemented if `parasite == "vivax"`

Reply:
Makes sense. I had a thought that at least the parameters for drug efficacy would need to be different. I guess we can talk about that later.